src/scenes/main_menu_scene.cpp

```diff
@@ -80,14 +93,29 @@ class MainMenu : public Scene {
                 GuiControl::TEXTBOX, GuiControlProperty::TEXT_ALIGNMENT,
                 GuiTextAlignment::TEXT_ALIGN_LEFT
             );
-            static char a[100] = "Text Box";
             if (GuiTextBox(m_button_rects[Button::CONNECT], a, 100, draw_textbox)) {
                 status_bar_text = a;
+                a[0] = 0;
                 draw_textbox = false;
             }
         } else if (m_button_pressed[Button::CONNECT]) {
             draw_textbox = true;
         }
+
+        float t = GetTime();
```

Comment:
where this function is defined?

Reply:
it is raylib function. should have called cpp version, but it is just a wrapper around c version......